src/styles/vars.scss

```
@@ -0,0 +1,7 @@
+$accent_color: #00acdc;
+$gray_elements: #f3f3f3;
+$primary-color: #060b35;
+$secondary_color: #616070;
+$white_color: #fff;
+$title_size: 12px;
+$desc_size: 10px;
```

It is better to use more generic variable names. 

```suggestion
$accent_color: #00acdc;
$second_color: #f3f3f3;
$primary_color: #060b35;
$secondary_color: #616070;
$button_color_active: #fff;
$title_size: 12px;
$description_size: 10px;
```